src/Knp/DictionaryBundle/Dictionary/CallableDictionary.php

```diff
@@ -21,14 +21,19 @@ class CallableDictionary implements Dictionary
      */
     private $callable;
 
+    /** @var array */
+    private $callableArgs;
+
     /**
      * @param string   $name
      * @param callable $callable
+     * @param array    $callableArgs
      */
-    public function __construct($name, callable $callable)
+    public function __construct($name, callable $callable, array $callableArgs = [])
     {
-        $this->name     = $name;
-        $this->callable = $callable;
+        $this->name         = $name;
+        $this->callable     = $callable;
+        $this->callableArgs = $callableArgs;
```

Comment:
indent

Reply:
To fix this, I assume we firstly need to change this: https://github.com/biig-io/DictionaryBundle/blob/master/.php_cs